demo/actions.py

```diff
@@ -485,3 +487,86 @@ def submit(self,
         dispatcher.utter_template('utter_restart_with_button', tracker)
 
         return [FollowupAction('action_listen')]
+
+
+class CommunityEventAction(Action):
+    """Utters Rasa community events."""
+
+    def __init__(self):
+        self.last_event_update = None
+        self.events = None
+        self.events = self._get_events()
+
+    def name(self) -> Text:
+        return "action_get_community_events"
+
+    def _get_events(self) -> List['CommunityEvent']:
+        if self.events is None or self._are_events_expired():
+            from demo.community_events import get_community_events
+            logger.debug("Getting events from website.")
+            self.last_event_update = datetime.now()
+            self.events = get_community_events()
+
+        return self.events
```

Comment:
I think something that would be useful in the `get_events()` function (or `get_community_events()` I guess), is a check that the events are actually in the future. on our website we have a lot of events that are in the past